pkg/services/searchV2/extract/dashboard.go

```diff
@@ -139,6 +227,35 @@ func ReadDashboard(stream io.Reader, lookup DatasourceLookup) (*DashboardInfo, e
 		}
 	}
 
+	for i, panel := range dash.Panels {
```

Comment:
Can you add a comment explaining what this does?  I *think* something like:
```
replace variable datasource references with concrete values
```

Reply:
extracted some fns